settings.py

```diff
@@ -116,7 +116,13 @@ def int_or_float(string):
 
 
 def _settings():
-    return sublime.load_settings("PackageDev.sublime-settings")
+    """Return global package settings object or load it first if required."""
```

Comment:
FYI, ST caches settings in its internal API already, which is why calling `.load_settings` repeatedly is acceptable, but I'm okay with the change. The objects were still being created for no real benefit.

```py
>>> sublime.load_settings("PackageDev.sublime-settings").settings_id == sublime.load_settings("PackageDev.sublime-settings").settings_id
True
```